vng_api_common/filters.py

```diff
@@ -73,6 +74,10 @@ def __init__(self, *args, **kwargs):
     def url_to_pk(self, url: str):
         parsed = urlparse(url)
         path = parsed.path
+
+        if path == "/":
+            path = ""
```

Comment:
Why is this needed?

Reply:
if I would try to filter on for instance `?catalogus=https://example.com/`, the path would be `"/"`, and this does not raise an `ObjectDoesNotExist`, but it when checking if the callback has a viewset (here: https://github.com/VNG-Realisatie/vng-api-common/blob/master/vng_api_common/utils.py#L61), causing an internal server error

I guess it is better to remove the trailing slash in `get_resource_for_path`